crucible/src/Lang/Crucible/Backend.hs

```diff
@@ -641,6 +641,11 @@ runCHC bak uninterp_inv_fns  = liftIO $ do
   implications <- proofObligationsAsImplications bak
   clearProofObligations bak
 
+  withFile "foo.smt2" WriteMode $ \handle ->
+    Z3.writeZ3HornSMT2File sym True handle uninterp_inv_fns implications
+  withFile "foo.sy" WriteMode $ \handle ->
+    CVC5.writeCVC5SyFile sym handle uninterp_inv_fns implications
```

Comment:
Currently, always creates these files which record the attempted solver operation for examination in case of failure.  Creating these should be configurable; used for debugging only.